models/model.articles.js

```diff
@@ -38,4 +38,31 @@ exports.formatArticles = () => {
       .then(({ rows }) => {
         return rows
       })
+}
+
+exports.getComments = (id) => {
+  return db.query(
+    `SELECT
+      comment_id,
+      votes,
+      created_at,
+      author,
+      body,
+      article_id
+    FROM
+      comments
+    WHERE
+      article_id = $1
+    ORDER BY
+      created_at DESC;`, [id])
+  .then(({ rows }) => {
+    const comments = rows
+    if (comments.length === 0) {
+      return Promise.reject({
+        status: 404,
+        msg: 'Article not found'
+      });
```

Comment:
Unfortunately, this won't work when we account for articles with no comments. You will need to do a separate check-articles-exist db.query.